css/main.css

```diff
@@ -682,28 +676,29 @@ hr {
     padding: 4%;
     background-color: #f5f5f5;
     text-align: center;
-}
-.logo-desciption h2{
-  text-align: center;
-}
+  }
 
-.about-hero-section {
-  background-image: url(../images/about_bg_01.png);
-  background-size: 100%;
-  background-color: #d3d3d3;
-  padding-bottom: 2%;
-  background-position: left;
-  background-repeat: repeat-x;
-}
+  .logo-desciption h2 {
+    text-align: center;
+  }
 
-.single-event {
-  width: 90%;
-  display: flex;
-  flex-direction: column;
-  justify-content: space-around;
-  flex-wrap: wrap;
-  align-items: center;
-  padding: 60px 20px;
-  margin: 10px;
-}
+  .about-hero-section {
+    background-image: url('../images/about_bg_01.png');
+    background-size: 100%;
+    background-color: #d3d3d3;
+    padding-bottom: 2%;
+    background-position: left;
+    background-repeat: repeat-x;
+  }
```

Comment:
- [ ] Please know that the background here isn't similar to the one in the original design, it's a lot darker, this is how it looks on your website:
  
  ![image](https://user-images.githubusercontent.com/20567503/196533685-5868c966-8862-4f3f-911d-2ae395608ed7.png)

  And this is how it looks on the original design:

  ![image](https://user-images.githubusercontent.com/20567503/196533734-8fe47c2e-8d24-4384-bba5-0dd3f9d7ef87.png)
  
  Kindly fix this so that it's similar to the original design. 😄